### PR TITLE
virtio_serial: Add test case for large file transfer

### PR DESCRIPTION
**Pull request by vivianQizhu**
id: 1588423
Signed-off-by: Qianqian Zhu <qizhu@redhat.com>

### CODE REVIEW

**Comment by vivianQizhu** [comment]
@nanliu-r Please help review if the steps match our test requirement, thanks.
 (1/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_host_to_guest: PASS (661.91 s)
 (2/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_guest_to_host: PASS (777.45 s)

File: qemu/tests/virtio_serial_large_file_transfer.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    5) Check guest running well, no crash
+
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env: Dictionary with test environment.
+    """
+
+    sender = params['file_sender']
+    file_size = int(params.get("filesize", 100))
+    repeat_time = int(params.get("repeat_time", 600))
```
Comment (nanliu-r) [inline qemu/tests/virtio_serial_large_file_transfer.py]: Seems there is no parameter 'repeat_time' in Dic params.

Comment (vivianQizhu) [reply qemu/tests/virtio_serial_large_file_transfer.py]: @nanliu-r Yes, so it utilizes the default value `600`, just leave an interface to customize the parameter, users can define it when needed.

**Comment by nanliu-r** [comment]
> @nanliu-r Please help review if the steps match our test requirement, thanks.
> (1/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_host_to_guest: PASS (661.91 s)
> (2/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_guest_to_host: PASS (777.45 s)

Yes, it matches test requirement and Test PASS. Thanks.

 (1/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_host_to_guest: PASS (654.87 s)
 (2/2) Host_RHEL.m8.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.1.0.x86_64.io-github-autotest-qemu.virtio_serial_file_transfer.tcp_socket.long_time_test.from_guest_to_host: PASS (1277.12 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB TIME   : 1933.92 s

**Comment by nanliu-r** [comment]
ACK.

**Pull request merged**